internal/update.camlp5

```diff
@@ -182,7 +170,7 @@ let print_same_name conf base p =
         (fun p ->
            Wserver.printf "<li>";
            begin match p with
```

Comment:
... and remove `let pl = restrict_to_small_list pl` ...
... and replace this `begin match ... end` with `print_person_parents_and_spouse conf base p` directly.